pkg/graph/resolvers/trb_request.go

```diff
@@ -334,3 +335,50 @@ func IsRecentTRBRequest(ctx context.Context, obj *models.TRBRequest, now time.Ti
 	recentIfAfterDate := now.AddDate(0, 0, numDaysToConsiderRecent)
 	return obj.CreatedAt.After(recentIfAfterDate)
 }
+
+// GetTRBLeadInfo retrieves the user info of a TRB request's lead
+func GetTRBLeadInfo(ctx context.Context, trbLead *string) (*models.UserInfo, error) {
+	var trbLeadInfo *models.UserInfo
+	if trbLead != nil {
+		info, err := dataloaders.GetUserInfo(ctx, *trbLead)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Might want to explicitly handle a case where we 

1. Assign a TRB Lead
2. That person leaves CMS (therefore they no longer resolve in LDAP)

We don't want the request to error here, but maybe do want to handle a case like this more gracefully? (i.e. creating a `UserInfo` constant object that represents an `Unassigned` user, like

```go
trbLeadInfo = &models.UserInfo{
  	CommonName: "Unknown"
	Email: "unknown@unknown.unknown",
	EuaUserID: "0000",
}
```

Might not be worth implementing here, but is something we should track as a potential case. If we want to retain lead info after someone leaves CMS, we might want to consider not saving EUAs in this field, and instead maybe just saving name/email directly? (Thought, that comes with it's own set of problems...)